### PR TITLE
add r-clustofvar

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Requirement r-pcamixdata in side [PR](https://github.com/conda-forge/staged-recipes/pull/13120)

Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/r-clustofvar/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  commands:
+    - $R -e "library('ClustOfVar')"           # [not win]
+    - "\"%R%\" -e \"library('ClustOfVar')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=ClustOfVar
+  license: GPL (>= 2.0)
+  summary: Cluster analysis of a set of variables. Variables can be quantitative, qualitative
+    or a mixture of both.
+  license_family: GPL3
```
Comment (dbast) [inline recipes/r-clustofvar/meta.yaml:43]:
```suggestion
  license_family: GPL3
  license_file:
    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```

File: recipes/r-clustofvar/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - r-pcamixdata
+
+test:
+  commands:
+    - $R -e "library('ClustOfVar')"           # [not win]
+    - "\"%R%\" -e \"library('ClustOfVar')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=ClustOfVar
+  license: GPL (>= 2.0)
```
Comment (dbast) [inline recipes/r-clustofvar/meta.yaml:40]:
```suggestion
  license: GPL-2.0-or-later
```

File: recipes/r-clustofvar/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  summary: Cluster analysis of a set of variables. Variables can be quantitative, qualitative
+    or a mixture of both.
+  license_family: GPL3
+
+extra:
+  recipe-maintainers:
+    - FernandezMathieu
```
Comment (dbast) [inline recipes/r-clustofvar/meta.yaml:47]:
```suggestion
    - conda-forge/r
    - FernandezMathieu
```

**Comment by dbast** [comment]
Requires r-pcamixdata to be added to conda-forge.

**Comment by FernandezMathieu** [comment]
@conda-forge-admin, please restart ci

**Comment by dbast** [comment]
Re-triggered too early ... Watch https://github.com/conda-forge/r-pcamixdata-feedstock ... CI finished successful + 40 minutes are required.

**Comment by FernandezMathieu** [comment]
@conda-forge-admin, please restart ci

**Pull request merged**